functions/src/functions/webhooks/auth/twilio-verify.ts

```diff
@@ -0,0 +1,97 @@
+import { HttpsError, onCall } from 'firebase-functions/v2/https';
+import { Twilio } from 'twilio';
+import { AuthAdmin } from '../../../../../shared/src/firebase/admin/AuthAdmin';
+import { TWILIO_ACCOUNT_SID, TWILIO_AUTH_TOKEN, TWILIO_VERIFY_SERVICE_SID } from '../../../config';
+
+type VerifyRequest = {
+	phoneNumber: string;
+	otp: string;
+};
+
+if (!TWILIO_ACCOUNT_SID || !TWILIO_AUTH_TOKEN || !TWILIO_VERIFY_SERVICE_SID) {
```

Comment:
You need to remove this here. Otherwise, the tests fail because this file is imported and these code lines get executed.